frontend/src/component/user/Login/Login.tsx

```diff
@@ -26,7 +27,8 @@ const Login = () => {
     const query = useQueryParams();
     const resetPassword = query.get('reset') === 'true';
     const invited = query.get('invited') === 'true';
-    const redirect = query.get('redirect') || '/';
+    const redirect =
```

Comment:
I believe we need to clear the parameter after a successful login. Session storage might be more suited as it is cleared on logout. Should we also consider a TTL?

Reply:
we clear the local storage on log out and we use it over session pretty consistently.  A short TTL makes sense since this is essentially aimed at holding this info until we get the callback from the auth providers

Reply:
Implemented both clear on successful login and ttl of 10'

Reply:
@andreas-unleash I think session storage is more correct here, the redirect will only apply for the session the user initialises with the url. With local storage we might get into weird scenarios even with a 10 minute TTL. For example what happens if you load a page with the redirect url and then you close the tab? If you open a new tab and go to unleash I expect you'll read local storage and be redirected.

Reply:
fair. I will change it